pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -115,6 +118,26 @@
      */
     public static final String DEFAULT_SOUND_FILENAME = "default";
 
+    /**
+     * The passive interruption level.
```

Comment:
For all of these levels, can we provide more specific docs? I think it's okay to quote the main docs as long as we provide attribution. For example:

```
According to <a href="...">Apple's documentation</a>, notifications with a {@code passive} interruption level contain "information people can view at their leisure, like a restaurant recommendation."
```

Reply:
The enum `InterruptionLevel` now reports Apple's HIG description for each level.